viz.yaml

```diff
@@ -192,6 +198,28 @@ publish:
       figure: map
       caption: states
   -
+    id: doy-section
```

Comment:
We probably want to include the doy and graph section in the same section if embed will be a thing